example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -35,6 +35,71 @@
 #error "unsupported CK_TILE_PIPELINE_DEFAULT value"
 #endif
 
+struct GemmBasicConfig
```

Comment:
Please rename to `GemmConfig` you might also change the file name to `gemm.hpp`.